octobot_trading/personal_data/positions/states/active_position_state.py

```diff
@@ -0,0 +1,30 @@
+#  Drakkar-Software OctoBot-Trading
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import octobot_trading.personal_data.positions.states.idle_position_state as idle_position_state
+
+
+class ActivePositionState(idle_position_state.IdlePositionState):
+    """
+    ActivePositionState is the state of a position that has a non-zero size
+    """
+    def is_active(self) -> bool:
+        """
+        :return: True if the Position has a non-zero size
+        """
+        return True
+
+    def _is_compatible_size(self):
+        return self.position.is_idle()
```

Comment:
Shouldn't it be `not self.position.is_idle()`?

Reply:
right !